service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -582,6 +582,10 @@ public OrderWithAddressesResponseDto saveCurrentAddressForOrder(CreateAddressReq
         address.setUser(currentUser);
         address.setActual(addresses.isEmpty());
         address.setAddressStatus(AddressStatus.NEW);
+
+        address.setDistrict(addressRequestDto.getDistrict());
+        address.setDistrictEn(addressRequestDto.getDistrictEn());
```

Comment:
please add test for the new functionality

Reply:
added